packages/pipeline-editor/src/TabbedPanelLayout/index.tsx

```diff
@@ -80,16 +80,17 @@ function TabbedPanelLayout({
             </div>
           ))}
         </div>
-        {experimental ? (
+        {showCloseButton === true && (
```

Comment:
Couldn't this just be `showCloseButton`?

Reply:
I prefer to be explicit, we’re using typescript so it doesn’t matter as much, but if showCloseButton wasn’t a boolean strange things can happen. !!showCloseButton would also work or using the ternary with null like before

Reply:
An example of this would be if showCloseButton was `0`. It’s falsy so it wouldn’t render the component, but it’s also a value so react would render the actual zero